lib/v2/screens/explore_screens/vote_screens/proposals/proposals_list.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/components/full_page_error_indicator.dart';
+import 'package:seeds/v2/components/full_page_loading_indicator.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/navigation/navigation_service.dart';
+import 'package:seeds/v2/screens/explore_screens/vote_screens/proposals/components/loading_indicator_list.dart';
+import 'package:seeds/v2/screens/explore_screens/vote_screens/proposals/viewmodels/bloc.dart';
+import 'package:seeds/v2/screens/explore_screens/vote_screens/vote/interactor/viewmodels/proposal_type_model.dart';
+
+class ProposalsList extends StatefulWidget {
+  final ProposalType proposalType;
+
+  const ProposalsList({Key? key, required this.proposalType}) : super(key: key);
+
+  @override
+  _ProposalsListState createState() => _ProposalsListState();
+}
+
+class _ProposalsListState extends State<ProposalsList> with AutomaticKeepAliveClientMixin {
+  late ProposalsListBloc _proposalsBloc;
+
+  @override
+  void initState() {
+    _proposalsBloc = ProposalsListBloc(widget.proposalType);
+    WidgetsBinding.instance?.addPostFrameCallback((_) {
+      if (!mounted) {
+        print('Screen not mounted --> call avoided.');
+        return;
+      }
+      _proposalsBloc.add(const InitialLoadProposals());
+    });
```

Comment:
Workaround to handle a tabs Flutter issue (https://github.com/flutter/flutter/issues/27680)